config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   resources :users, only: [:new, :create, :show, :edit, :update, :destroy]
   resources :budgets, only: [:new]
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
```

Comment:
Are these two routes really needed? or could you use something like:
`resources :sessions, only: [:new, :create, :destroy]`
and then use the `new_session_path`...

Reply:
Again, old habits, I will delete them